client/me/purchases/concierge-banner/index.jsx

```diff
@@ -42,86 +42,91 @@ class ConciergeBanner extends Component {
 	getBannerContent() {
 		const { bannerType, translate } = this.props;
 
-		let headerText;
-		let mainText;
+		let title;
+		let description;
 		let buttonText;
-		let buttonHref;
-		let illustrationUrl;
+		let event;
 
 		switch ( bannerType ) {
 			case CONCIERGE_HAS_UPCOMING_APPOINTMENT:
-				headerText = translate( 'Your appointment is coming up!' );
-				mainText = translate(
-					'Get ready with your questions for your upcoming Quick Start session appointment.',
+				title = translate( 'Your session is coming up soon' );
+				description = translate(
+					'Your {{supportLink}}Quick Start support session{{/supportLink}} is approaching. Get ready for your one-to-one with our Happiness Engineer.',
 					{
-						comment:
-							'Quick Start Session is a one-on-one video session between the user and our support staff.',
+						components: {
+							supportLink: (
+								<a
+									target={ '_blank' }
+									rel={ 'noreferrer' }
+									href="https://wordpress.com/discover-wordpress/2019/03/21/getting-the-most-out-of-our-business-concierge-service/"
+								/>
+							),
+						},
 					}
 				);
 				buttonText = translate( 'Session dashboard' );
-				buttonHref = '/me/quickstart';
-				illustrationUrl = conciergeImage;
+				event = 'view-concierge-dashboard';
 				break;
-
+			case CONCIERGE_HAS_AVAILABLE_PURCHASED_SESSION:
 			case CONCIERGE_HAS_AVAILABLE_INCLUDED_SESSION:
```

Comment:
Might it be worth creating a dedicated constant to represent `CONCIERGE_HAS_AVAILABLE_SESSION`? We may or may not be able to collapse the usages, but it would be simpler if we aren't distinguishing between these two cases any more.